src/docs/product/relay/options.mdx

```diff
@@ -126,3 +126,9 @@ default.
 | -------------- | ------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | sentry.enabled | Boolean | default: `false` | Determines whether to report internal errors to a separate DSN. `false` means that no internal errors will be sent, but will still be logged.                                        |
 | sentry.dsn     | String  | optional: `true` | Sentry DSN to which you can report internal Relay failures. We recommend setting this to a value that sends Relay errors directly to Sentry instead of sending them to itself first. |
+
+## Geoip
+
+| key        | type   | value           | description                                                                                                              |
+| ---------- | ------ | --------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| geoip.path | String | default: `null` | The path to the MaxMind GeoIP City database file. When set the additional GeoIp information will be added to the events. |
```

Comment:
```suggestion
| geoip.path | String | default: `null` | The path to the MaxMind GeoIP City database file. When set the additional GeoIP information will be added to the events. |
```

Does it need to be a MaxMind database? Would other sources work?

Reply:
yes, that's what we currently support.